internal/pkg/skuba/cni/cilium.go

```diff
@@ -113,6 +113,8 @@ func CreateCiliumSecret(client clientset.Interface, ciliumVersion string) error
 			"ca.crt":            pkiutil.EncodeCertPEM(caCert),
 		}
 	case strings.HasPrefix(ciliumVersion, "1.6"):
+		fallthrough
+	case strings.HasPrefix(ciliumVersion, "1.7"):
```

Comment:
I'm wondering if we can maybe just handle that in the `default:` case, to not end up with multiple `strings.HasPrefix("1.x")` cases. But we can do that in a follow-up PR.